### ISSUE TITLE
Add metric and loss that take each output of Helixer into account

### ISSUE THREAD

**Issue by soi** [weberlab-hhu/GeenuFF#48]

**Comment by soi** [comment]
Metrics are implemented and loss weights can be done with model.fit(class_weights=)
